src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -611,7 +611,7 @@ public DarCollection cancelDarCollectionElectionsAsAdmin(DarCollection collectio
     // Cancel all DAR elections
     cancelElectionsForReferenceIds(referenceIds);
 
-    return darCollectionDAO.findDARCollectionByCollectionId(collection.getDarCollectionId());
+    return getByCollectionId(collection.getDarCollectionId());
```

Comment:
This ensures that datasets are populated so we can log dar cancellations for the admin case